src/core/manager.c

```diff
@@ -4383,6 +4383,7 @@ char *manager_taint_string(Manager *m) {
         assert(m);
 
         buf = new(char, sizeof("split-usr:"
+                               "unmerged-usr:"
```

Comment:
Let's merge #22993 first and rebase this on top. (Since it needs updating anyway.)